### DISCUSSION TITLE
The **Explore repositories**  are frequent duplicated

### DISCUSSION THREAD

**Discussion by ice-tong** [community/community#68183] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The **Explore repositories**  are frequent duplicated, is this as expected?

- Case1
<img src="https://github.com/community/community/assets/32220263/f334191c-08ef-4ef4-9766-9207319dc2ea" height=300>

- Case2
<img src="https://github.com/community/community/assets/32220263/6af2d226-9434-49f3-a8f1-41323d6a4fad" height=300>

- Case3
<img src="https://github.com/community/community/assets/32220263/87e03cea-de44-4ea8-b744-e1b075c7e10c" height=300>

**Comment by SafeArrow816** [comment]
Could we reading about license ?
